docs/docs/identity-providers/readme.md

```diff
@@ -8,23 +8,14 @@ description: >-
 
 # Identity Provider Configuration
 
-This article describes how to configure Pomerium to use a third-party identity service for single-sign-on.
+Pomerium helps provide a zero-trust security model by verifying that every request to your upstream applications are from an authenticated and authorized user. User and group information is sourced from your Identity Provider (**IdP**). Pomerium communicates with IdPs following the [OpenID Connect][openid connect] (**OIDC**) standard for user identity, and a service account and/or API token for group/directory information. The steps for integrating Pomerium with an IdP are specific to each provider, but they generally share the same base requirements:
```

Comment:
```suggestion
Pomerium helps provide a zero-trust security model by verifying that every request to your upstream applications is from an authenticated and authorized user. User and group information is sourced from your Identity Provider (**IdP**). Pomerium communicates with IdPs following the [OpenID Connect][openid connect] (**OIDC**) standard for user identity, and a service account and/or API token for group/directory information. The steps for integrating Pomerium with an IdP are specific to each provider, but they generally share the same base requirements:
```

Reply:
Pomerium provides single-sign-on authentication and user identity details by integrating with your downstream identity provider of choice. That authentication integration is achieved using OAuth2, and OIDC. Where available, Pomerium also supports pulling additional data (like groups) using directory synchronization. An additional API token is required for directory sync.